aio/deploy/helm-chart/kubernetes-dashboard/templates/psp.yaml

```
@@ -0,0 +1,72 @@
+#f .Values.podSecurityPolicy.enabled -}} Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{ if .Values.podSecurityPolicy.enabled -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "kubernetes-dashboard.fullname" . }}-psp
+  labels:
+    app: {{ template "kubernetes-dashboard.fullname" . }}
```

You can use our labels template here to unify:
```suggestion
{{ include "kubernetes-dashboard.labels" . | indent 4 }}
```